package.json

```diff
@@ -14,7 +14,7 @@
   "repository": "https://github.com/ember-cli/ember-fetch",
   "scripts": {
     "build": "ember build",
-    "lint:js": "eslint ./*.js addon addon-test-support app config lib server test-support tests",
+    "lint:js": "eslint . --cache",
```

Comment:
Nice! But we should `.eslintcache` to `.gitignore` then, I guess?

Reply:
you're totally right, I forgot that 🤦‍♂️ 

fixed!